server/container/guice/cassandra-rabbitmq-ldap-guice/src/test/resources/ldif-files/populate.ldif

```diff
@@ -0,0 +1,11 @@
+dn: ou=people, dc=james,dc=org
```

Comment:
Given that we import LdapTestExtension from cassandra-ldap-guice mvn module, do we really need to duplicate this LDIF and docker file?